components/global/core/io.cellery.observability.k8s.client/src/main/java/io/cellery/observability/k8s/client/Constants.java

```diff
@@ -23,6 +23,7 @@
  */
 public class Constants {
     public static final String NAMESPACE = "default";
+    public static final String NODE_NAME = "node1";
```

Comment:
Shall we move this to BaseTestCase ? Since this is a test related constant ?

Reply:
Fixed. I had thought this constants are test constants, my bad. Now, I moved into BaseTestCase.